src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -0,0 +1,279 @@
+            self.argvals["fluent_debug"] = True
+
+        server_info_file_name = _get_server_info_file_name()
+        launch_string = _generate_launch_string(
+            self.argvals,
+            self.meshing_mode,
+            self.show_gui,
+            self.additional_arguments,
+            server_info_file_name,
+        )
+
+        sifile_last_mtime = Path(server_info_file_name).stat().st_mtime
+        if self.env is None:
+            setattr(self, "env", {})
+        kwargs = _get_subprocess_kwargs_for_fluent(self.env)
+        if self.cwd:
+            kwargs.update(cwd=self.cwd)
+        launch_string += _build_journal_argument(self.topy, self.journal_file_names)
+
+        if _is_windows():
+            # Using 'start.exe' is better, otherwise Fluent is more susceptible to bad termination attempts
+            launch_cmd = 'start "" ' + launch_string
+        else:
+            launch_cmd = launch_string
+
+        try:
+            logger.debug(f"Launching Fluent with command: {launch_cmd}")
+
+            subprocess.Popen(launch_cmd, **kwargs)
+
+            try:
+                _await_fluent_launch(
+                    server_info_file_name, self.start_timeout, sifile_last_mtime
+                )
+            except TimeoutError as ex:
+                if _is_windows():
+                    logger.warning(f"Exception caught - {type(ex).__name__}: {ex}")
+                    launch_cmd = launch_string.replace('"', "", 2)
+                    kwargs.update(shell=False)
+                    logger.warning(
+                        f"Retrying Fluent launch with less robust command: {launch_cmd}"
+                    )
+                    subprocess.Popen(launch_cmd, **kwargs)
+                    _await_fluent_launch(
+                        server_info_file_name, self.start_timeout, sifile_last_mtime
+                    )
+                else:
+                    raise ex
+
+            session = self.new_session.create_from_server_info_file(
+                server_info_file_name=server_info_file_name,
+                remote_file_handler=RemoteFileHandler(),
+                cleanup_on_exit=self.cleanup_on_exit,
+                start_transcript=self.start_transcript,
+                launcher_args=self.argvals,
+                inside_container=False,
+            )
+            start_watchdog = _confirm_watchdog_start(
+                self.start_watchdog, self.cleanup_on_exit, session.fluent_connection
+            )
+            if start_watchdog:
+                logger.info("Launching Watchdog for local Fluent client...")
+                ip, port, password = _get_server_info(server_info_file_name)
+                watchdog.launch(os.getpid(), port, password, ip)
+            if self.case_file_name:
+                if self.meshing_mode:
+                    session.tui.file.read_case(self.case_file_name)
+                else:
+                    session.read_case(self.case_file_name, self.lightweight_mode)
```

Comment:
@ansjsia @raph-luc the old version was changed here to this newer version.
I.e. from 
```
elif lightweight_mode:
                    session.read_case_lightweight(case_file_name)
```